examples/basic/adapter.ts

```diff
@@ -20,27 +20,24 @@ function trackClient(client: Client) {
 			adapters.get(payload.guild_id)?.onVoiceStateUpdate(payload);
 		}
 	});
-}
-
-const trackedGuilds = new Map<WebSocketShard, Set<Snowflake>>();
-
-function cleanupGuilds(shard: WebSocketShard) {
-	const guilds = trackedGuilds.get(shard);
-	if (guilds) {
-		for (const guildId of guilds.values()) {
-			adapters.get(guildId)?.destroy();
+	client.on(Constants.Events.SHARD_DISCONNECT, (_, shardID) => {
```

Comment:
If you're going to call the guild's ID with `Id` (see line 26), then change this to `shardId`, otherwise rename the variable below to `guildID`.